chains/astar-sepolia-testnet.json

```diff
@@ -16,10 +16,6 @@
     {
       "alias": "default",
       "rpcUrl": "https://rpc.startale.com/zkyoto"
```

Comment:
Maybe we can leverage `https://astar-zkyoto-rpc.dwellir.com` which is listed in their [docs](https://docs.astar.network/docs/build/zkEVM/quickstart/#connecting-to-zkevm). I'm meaning something like:

```json
    {
      "alias": "dwellir-public",
      "rpcUrl": "https://astar-zkyoto-rpc.dwellir.com"
    }
```